src/lib/shell/MainLoopEFR32.cpp

```diff
@@ -135,7 +135,9 @@ int TokenizeLine(char * buffer, char ** tokens, int max_tokens)
         else if (IsSeparator(buffer[i]))
         {
             buffer[i] = 0;
-            if (!IsSeparator(buffer[i + 1]))
+            // Don't treat the previous character as a separator if this one is 0
+            // otherwise the trailing space will become a token
+            if (!IsSeparator(buffer[i + 1]) && buffer[i + 1] != 0)
```

Comment:
```suggestion
            if (!IsSeparator(buffer[i + 1]) && (buffer[i + 1] != 0))
```

Reply:
I wholeheartedly agree with the suggestion, not sure why I didn't put the parenthesis from the start. However, I would really prefer not to re-spin the PR given how flaky the CI has been lately.